symbolicationWebService.py

```diff
@@ -146,50 +167,57 @@ def ReadConfigFile():
   if len(sys.argv) == 1:
     return True
   elif len(sys.argv) > 2:
-    LogError("Usage: symbolicationWebService.py [<config file>]")
+    logging.error("Usage: symbolicationWebService.py [<config file>]")
     return False
   elif len(sys.argv) == 2:
     try:
-      configParser = ConfigParser.ConfigParser()
-      # Make parser case-sensitive
-      configParser.optionxform=str
+      # ConfigParser uses the pattern %(<variable>)<type> for variable substitution,
+      # so '%' found in environment variable values will raise an error. We replace
+      # '%' by '%%' to make the parser understand it is literal character.
+      environ = {key:value.replace(r"%", r"%%") for key, value in os.environ.iteritems()}
+
+      configParser = CaseSensitiveConfigParser(environ)
       configFile = open(sys.argv[1], "r")
       configParser.readfp(configFile)
       configFile.close()
     except ConfigParser.Error as e:
-      LogError("Unable to parse config file " + sys.argv[1] + ": " + str(e))
-    except:
-      LogError("Unable to open config file " + sys.argv[1])
+      logging.error("Unable to parse config file %s: %s", sys.argv[1], e)
```

Comment:
Logging has not been configured yet, so a call to `LogError` will raise an exception.